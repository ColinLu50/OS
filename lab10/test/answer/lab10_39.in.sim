14533
490
490
261
490
260
