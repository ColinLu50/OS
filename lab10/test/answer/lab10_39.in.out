FCFS
228 -> 31 -> 230 -> 187 -> 38 -> 238 -> 26 -> 162 -> 84 -> 36 -> 262 -> 42 -> 145 -> 251 -> 170 -> 159 -> 5 -> 71 -> 77 -> 68 -> 194 -> 71 -> 36 -> 167 -> 218 -> 102 -> 226 -> 126 -> 232 -> 180 -> 252 -> 40 -> 101 -> 75 -> 101 -> 249 -> 171 -> 238 -> 24 -> 69 -> 10 -> 170 -> 22 -> 63 -> 231 -> 233 -> 105 -> 199 -> 154 -> 243 -> 106 -> 39 -> 218 -> 90 -> 166 -> 239 -> 102 -> 65 -> 168 -> 204 -> 91 -> 159 -> 1 -> 51 -> 193 -> 177 -> 234 -> 21 -> 107 -> 87 -> 182 -> 9 -> 106 -> 249 -> 16 -> 27 -> 22 -> 111 -> 220 -> 68 -> 112 -> 20 -> 186 -> 0 -> 223 -> 58 -> 109 -> 70 -> 51 -> 54 -> 73 -> 110 -> 130 -> 107 -> 162 -> 235 -> 242 -> 121 -> 112 -> 140 -> 160 -> 57 -> 247 -> 86 -> 108 -> 48 -> 166 -> 187 -> 200 -> 12 -> 102 -> 198 -> 123 -> 18 -> 243 -> 242 -> 234 -> 53 -> 143 -> 155 -> 39 -> 242 -> 8 -> 227 -> 118 -> 99 -> 85 -> 55 -> 141 -> 131 -> 43 -> 27 -> 101 -> 169 -> 111 -> 216 -> 171 -> 77 -> 121 -> 9 -> 236 -> 26 -> 119 -> 140 -> 144 -> 56 -> 115 -> 45 -> 250 -> 232 -> 15 -> 185 -> 31
14533
SSTF
228 -> 227 -> 226 -> 223 -> 220 -> 218 -> 218 -> 216 -> 204 -> 200 -> 199 -> 198 -> 194 -> 193 -> 187 -> 187 -> 186 -> 185 -> 182 -> 180 -> 177 -> 171 -> 171 -> 170 -> 170 -> 169 -> 168 -> 167 -> 166 -> 166 -> 162 -> 162 -> 160 -> 159 -> 159 -> 155 -> 154 -> 145 -> 144 -> 143 -> 141 -> 140 -> 140 -> 131 -> 130 -> 126 -> 123 -> 121 -> 121 -> 119 -> 118 -> 115 -> 112 -> 112 -> 111 -> 111 -> 110 -> 109 -> 108 -> 107 -> 107 -> 106 -> 106 -> 105 -> 102 -> 102 -> 102 -> 101 -> 101 -> 101 -> 99 -> 91 -> 90 -> 87 -> 86 -> 85 -> 84 -> 77 -> 77 -> 75 -> 73 -> 71 -> 71 -> 70 -> 69 -> 68 -> 68 -> 65 -> 63 -> 58 -> 57 -> 56 -> 55 -> 54 -> 53 -> 51 -> 51 -> 48 -> 45 -> 43 -> 42 -> 40 -> 39 -> 39 -> 38 -> 36 -> 36 -> 31 -> 31 -> 27 -> 27 -> 26 -> 26 -> 24 -> 22 -> 22 -> 21 -> 20 -> 18 -> 16 -> 15 -> 12 -> 10 -> 9 -> 9 -> 8 -> 5 -> 1 -> 0 -> 230 -> 231 -> 232 -> 232 -> 233 -> 234 -> 234 -> 235 -> 236 -> 238 -> 238 -> 239 -> 242 -> 242 -> 242 -> 243 -> 243 -> 247 -> 249 -> 249 -> 250 -> 251 -> 252 -> 262
490
SCAN
228 -> 227 -> 226 -> 223 -> 220 -> 218 -> 216 -> 204 -> 200 -> 199 -> 198 -> 194 -> 193 -> 187 -> 186 -> 185 -> 182 -> 180 -> 177 -> 171 -> 170 -> 169 -> 168 -> 167 -> 166 -> 162 -> 160 -> 159 -> 155 -> 154 -> 145 -> 144 -> 143 -> 141 -> 140 -> 131 -> 130 -> 126 -> 123 -> 121 -> 119 -> 118 -> 115 -> 112 -> 111 -> 110 -> 109 -> 108 -> 107 -> 106 -> 105 -> 102 -> 101 -> 99 -> 91 -> 90 -> 87 -> 86 -> 85 -> 84 -> 77 -> 75 -> 73 -> 71 -> 70 -> 69 -> 68 -> 65 -> 63 -> 58 -> 57 -> 56 -> 55 -> 54 -> 53 -> 51 -> 48 -> 45 -> 43 -> 42 -> 40 -> 39 -> 38 -> 36 -> 31 -> 27 -> 26 -> 24 -> 22 -> 21 -> 20 -> 18 -> 16 -> 15 -> 12 -> 10 -> 9 -> 8 -> 5 -> 1 -> 0 -> 0 -> 230 -> 231 -> 232 -> 233 -> 234 -> 235 -> 236 -> 238 -> 239 -> 242 -> 243 -> 247 -> 249 -> 250 -> 251 -> 252 -> 262
490
C-SCAN
228 -> 227 -> 226 -> 223 -> 220 -> 218 -> 216 -> 204 -> 200 -> 199 -> 198 -> 194 -> 193 -> 187 -> 186 -> 185 -> 182 -> 180 -> 177 -> 171 -> 170 -> 169 -> 168 -> 167 -> 166 -> 162 -> 160 -> 159 -> 155 -> 154 -> 145 -> 144 -> 143 -> 141 -> 140 -> 131 -> 130 -> 126 -> 123 -> 121 -> 119 -> 118 -> 115 -> 112 -> 111 -> 110 -> 109 -> 108 -> 107 -> 106 -> 105 -> 102 -> 101 -> 99 -> 91 -> 90 -> 87 -> 86 -> 85 -> 84 -> 77 -> 75 -> 73 -> 71 -> 70 -> 69 -> 68 -> 65 -> 63 -> 58 -> 57 -> 56 -> 55 -> 54 -> 53 -> 51 -> 48 -> 45 -> 43 -> 42 -> 40 -> 39 -> 38 -> 36 -> 31 -> 27 -> 26 -> 24 -> 22 -> 21 -> 20 -> 18 -> 16 -> 15 -> 12 -> 10 -> 9 -> 8 -> 5 -> 1 -> 0 -> 0 -> 263 -> 262 -> 252 -> 251 -> 250 -> 249 -> 247 -> 243 -> 242 -> 239 -> 238 -> 236 -> 235 -> 234 -> 233 -> 232 -> 231 -> 230
261
LOOK
228 -> 227 -> 226 -> 223 -> 220 -> 218 -> 216 -> 204 -> 200 -> 199 -> 198 -> 194 -> 193 -> 187 -> 186 -> 185 -> 182 -> 180 -> 177 -> 171 -> 170 -> 169 -> 168 -> 167 -> 166 -> 162 -> 160 -> 159 -> 155 -> 154 -> 145 -> 144 -> 143 -> 141 -> 140 -> 131 -> 130 -> 126 -> 123 -> 121 -> 119 -> 118 -> 115 -> 112 -> 111 -> 110 -> 109 -> 108 -> 107 -> 106 -> 105 -> 102 -> 101 -> 99 -> 91 -> 90 -> 87 -> 86 -> 85 -> 84 -> 77 -> 75 -> 73 -> 71 -> 70 -> 69 -> 68 -> 65 -> 63 -> 58 -> 57 -> 56 -> 55 -> 54 -> 53 -> 51 -> 48 -> 45 -> 43 -> 42 -> 40 -> 39 -> 38 -> 36 -> 31 -> 27 -> 26 -> 24 -> 22 -> 21 -> 20 -> 18 -> 16 -> 15 -> 12 -> 10 -> 9 -> 8 -> 5 -> 1 -> 0 -> 230 -> 231 -> 232 -> 233 -> 234 -> 235 -> 236 -> 238 -> 239 -> 242 -> 243 -> 247 -> 249 -> 250 -> 251 -> 252 -> 262
490
C-LOOK
228 -> 227 -> 226 -> 223 -> 220 -> 218 -> 216 -> 204 -> 200 -> 199 -> 198 -> 194 -> 193 -> 187 -> 186 -> 185 -> 182 -> 180 -> 177 -> 171 -> 170 -> 169 -> 168 -> 167 -> 166 -> 162 -> 160 -> 159 -> 155 -> 154 -> 145 -> 144 -> 143 -> 141 -> 140 -> 131 -> 130 -> 126 -> 123 -> 121 -> 119 -> 118 -> 115 -> 112 -> 111 -> 110 -> 109 -> 108 -> 107 -> 106 -> 105 -> 102 -> 101 -> 99 -> 91 -> 90 -> 87 -> 86 -> 85 -> 84 -> 77 -> 75 -> 73 -> 71 -> 70 -> 69 -> 68 -> 65 -> 63 -> 58 -> 57 -> 56 -> 55 -> 54 -> 53 -> 51 -> 48 -> 45 -> 43 -> 42 -> 40 -> 39 -> 38 -> 36 -> 31 -> 27 -> 26 -> 24 -> 22 -> 21 -> 20 -> 18 -> 16 -> 15 -> 12 -> 10 -> 9 -> 8 -> 5 -> 1 -> 0 -> 262 -> 252 -> 251 -> 250 -> 249 -> 247 -> 243 -> 242 -> 239 -> 238 -> 236 -> 235 -> 234 -> 233 -> 232 -> 231 -> 230
260
